### ISSUE TITLE
Enable tags inside embeds

### ISSUE THREAD

**Issue by tchavei** [Aliucord/plugin-requests#602]
### Discord Account

[root＠tchavei ~]＃#1337

### Plugin Description

We all go through this. We query a bot about an user and get a response beautifully formated inside an embed. Unfortunately, android only supports title and footer long tap to copy and the user or channel tags are totally inaccessible inside the embed body. There's already a plugin out there that helps to force unresolved tags to display properly but this would be one step further, actually being practical and useful for most users. 

### Go into more detail...

Basically, the plugin should allow tap on tags inside embeds and treat them as if they were inside the message body i.e. Bring up the user profile menu if a user tag or take us to the tagged channel if a channel tag.
Supporting embed body copy would be a plus but maybe that's material for another separate module? 

### Request Agreement

- [X] I did indeed check to make sure my plugin request is original!

**Comment by Vendicated** [comment]
Exists lol, ClickableMentions

**Comment by AarifZ** [comment]
Would be cool
